numpy/f2py/tests/src/isocintrin/isoCtests.f90

```diff
@@ -14,4 +14,21 @@ function wat(x, y) result(z) bind(c)
 
           z = x + 7
       end function wat
+      ! gh-25207
+      subroutine c_add_int64(a, b, c) bind(c)
+        integer(c_int64_t), intent(in) :: a, b
+        integer(c_int64_t), intent(out) :: c
+        c = a + b
+      end subroutine c_add_int64
+      ! gh-25207
+      subroutine add_arr(A, B, C)
+         integer(c_int64_t), intent(in) :: A(3)
+         integer(c_int64_t), intent(in) :: B(3)
+         integer(c_int64_t), intent(out) :: C(3)
+         integer :: j
+
+         do j = 1, 3
+            C(j) = A(j)+B(j)
+         end do
+      end subroutine
```

Comment:
Since we now want to test over a range of these, perhaps putting one of these, like `c_add_int64` into the `.py` file like `test_character` does.